.github/workflows/pr.yml

```diff
@@ -51,6 +83,7 @@ jobs:
     if: always()
     steps:
       - run: sh -c ${{
-          needs.build.result == 'success' &&
-          needs.lint.result == 'success' &&
-          needs.test.result == 'success' }}
+          (needs.build.result == 'success' || needs.build.result == 'skipped') &&
+          (needs.lint.result == 'success' || needs.lint.result == 'skipped') &&
+          (needs.test.result == 'success' || needs.test.result == 'skipped') && 
+          (needs.e2e.result == 'success' || needs.e2e.result == 'skipped') }}
```

Comment:
nit: who doesn't put a newline at the end of the file?  What are we, barbarians? 😉

Reply:
newline added!